stream.py

```
@@ -0,0 +1,59 @@
+import time
+
+from typing import Dict, Any
+from util import make_safe_number
+
+# CONSTANTS
+ORIGIN_STRING = "SDEC-API"
+STREAM_VERSION_STRING = "1.0"
+
+
+class Stream:
+    def __init__(self):
```

Ruff (0.15.21)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)